### ISSUE TITLE
About GetBatch

### ISSUE THREAD

**Issue by hktxt** [vlfeat/matconvnet#1188]
Actully, I have audio dataset files which I already write their path and labels in txt file. Then, I need to read audios from this txt file.
But, how do I write a GetBatch function to generate batchs and feed into the CNN? 
I just get to the MatConvNet for the first time. Like the tensorflow and pytorch can proceed training set as a iterator, which can feed into CNN sequentially. Does there exist some similar method in MatConvNet?
can anyone help?
